api/camunda/camunda-openapi.yaml

```diff
@@ -5550,6 +6330,48 @@ components:
         tenantId:
           description: The ID of the tenant that owns the job
           type: string
+    ActivatedJobNumberKeys:
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/ActivatedJobBase"
+      properties:
+        jobKey:
+          description: the key, a unique identifier for the job
+          type: integer
+          format: int64
+        processInstanceKey:
+          description: the job's process instance key
+          type: integer
+          format: int64
+        processDefinitionKey:
+          description: the key of the job's process definition
+          type: integer
+          format: int64
+        elementInstanceKey:
+          description: >
+            the unique key identifying the associated task, unique within the scope of the
+            process instance
+          type: integer
+          format: int64
+    ActivatedJob:
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/ActivatedJobBase"
+      properties:
+        jobKey:
+          description: the key, a unique identifier for the job
+          type: string
+        processInstanceKey:
+          description: the job's process instance key
+          type: string
+        processDefinitionKey:
+          description: the key of the job's process definition
+          type: string
+        elementInstanceKey:
+          description: >
+            the unique key identifying the associated task, unique within the scope of the
+            process instance
```

Comment:
TODO: Sentence case for all these descriptions. I'll push a PR upstream for this.

Reply:
TODONE in https://github.com/camunda/camunda/pull/26305